middleware/legacy/rules/static-pages.js

```diff
@@ -3,34 +3,52 @@
 import escapeRegExp from 'lodash/escapeRegExp';
 
 const redirects = {
-  '/collections/world-war-I': { path: '/entity/topic/83-1914-1918' },
-  '/collections/archaeology': { path: '/entity/topic/80-archaeology' },
-  '/collections/art': { path: '/entity/topic/190-art' },
+  '/about': '/about-us',
+  '/collections/archaeology': '/entity/topic/80-archaeology',
+  '/collections/art': '/entity/topic/190-art',
   '/collections/fashion': { path: '/search', query: { query: '', theme: 'fashion' } },
-  '/collections/industrial-heritage': { path: '/entity/topic/129-industrial-heritage' },
-  '/collections/manuscripts': { path: '/entity/topic/17-manuscripts' },
-  '/collections/maps': { path: '/entity/topic/151-maps-and-geography' },
-  '/collections/migration': { path: '/entity/topic/128-migration' },
-  '/collections/music': { path: '/entity/topic/62-music' },
-  '/collections/natural-history': { path: '/entity/topic/156-natural-history' },
-  '/collections/newspapers': { path: '/entity/topic/18-newspaper' },
-  '/collections/photography': { path: '/entity/topic/48-photography' },
-  '/collections/sport': { path: '/entity/topic/114-sport' },
-  '/exhibitions/foyer': { path: '/exhibitions' },
-  '/explore/galleries': { path: '/galleries' }
+  '/collections/industrial-heritage': '/entity/topic/129-industrial-heritage',
+  '/collections/manuscripts': '/entity/topic/17-manuscripts',
+  '/collections/maps': '/entity/topic/151-maps-and-geography',
+  '/collections/migration': '/entity/topic/128-migration',
+  '/collections/migration/collection-days-stories.html': '/migration/explore-migration-stories-from-events-in-these-cities',
+  '/collections/migration/collection-days': '/europeana-migration-collection-days',
```

Comment:
this one works, but only when visiting: '[host]/portal/en/collections/migration/explore-migration-stories', realistically people would be coming from 'https://www.europeana.eu/portal/en/collections/migration/explore-migration-stories.html' which icludes 'html'